src/interaction/ConstrainCOM.hpp

```diff
@@ -71,21 +71,19 @@ namespace espressopp {
 	    }
 	    
 	    Real3D _computeForce(Real3D diff,
-				 long unsigned int N_Constrain) {
+				 long unsigned int Total_mass) {
```

Comment:
Yes. Exactly. This is just mistake. I modified `long unsigned int` to `real`. Thanks you.